frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import * as classNames from 'classnames';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { K8sResourceKind } from '@console/internal/module/k8s/index';
+import { useFlag } from '@console/shared';
+
+import { StorageClusterKind } from '../../../../types';
+import { ocsResource } from '../../../../resources';
+import { GUARDED_FEATURES } from '../../../../features';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
+  ({ title, isLoading, error, className, children }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactElement;
+
+    if (isLoading) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error) {
+      status = (
+        <span className="co-dashboard-text--small text-muted">
+          {t('ceph-storage-plugin~Not available')}
+        </span>
+      );
+    } else {
+      status = <span className="ceph-storage-efficiency-card__item-text">{children}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__item-title">{title}</div>
+        <div
+          className={classNames('ceph-storage-efficiency-card__item-status', className)}
+          data-test={`${title}-mirroring-card-status`}
+        >
+          {status}
+        </div>
+      </div>
+    );
+  },
+);
+
+const MirroringCard: React.FC<DashboardItemProps> = ({
+  watchK8sResource,
+  stopWatchK8sResource,
+  resources,
+}) => {
+  const { t } = useTranslation();
+
+  /** to-do: add a better way to filter cards based on the flag in ocs-system-dashboard.tsx file.
+   * Right now even if we are returning null (when flag is set) from this component,
+   * there is an extra spacing between cards because of the GridItem FC.
+   */
```

Comment:
^^ Extra Space between Inventory and Storage Efficiency Cards...
![Screenshot from 2021-07-23 20-25-28](https://user-images.githubusercontent.com/39404641/126811938-d5242421-e31a-4637-b2cf-be3d0e26e819.png)